charts/portainer/values.yaml

```diff
@@ -10,12 +10,12 @@ enterpriseEdition:
   image:
     repository: portainer/portainer-ee
     tag: 2.0.0
```

Comment:
Shouldn't the tag value be updated as well?

Reply:
@ncresswell explained that we don't _have_ a 2.0.1 release of EE yet?

Reply:
My bad this is EE related. You can ignore this comment.